documentation/managing_modules.md

```diff
@@ -0,0 +1,183 @@
+# 🧪 Managing modules in Bolt projects
+
+Bolt version 2.30.0 introduced module workflows for Bolt projects that improve
+dependency management and make it easier to add new modules to your Bolt
+project. These workflows, which include the `bolt module install`, `bolt module
+add`, and `bolt project migrate` commands, are experimental.
+
+## What's changed?
```

Comment:
I think this should go after "How it works" so the user has a bit more context for these changes.

Reply:
Hm, I kind of disagree. I think most users reading this will already use Bolt and will be most interested this section, and it helps define the scope of what's changed so that reading on feels less intimidating.

Reply:
I actually originally had this section after How it works, but I didn't like how it separated "How it works" from "opting in" etc.